packages/network/src/createRelayStream.ts

```diff
@@ -22,7 +22,12 @@ export async function createRelayStream(signer: Signer, url: string, id: string)
   );
 
   // Signature that should be used to prove identity
-  const signature = { signature: await signer.signMessage("ecs-relay-service") };
+  // Set the signature to expire in 5000 ms
+  const signatureExpirationTime: number = Date.now() + 5000;
+  const signature = {
+    signature: await signer.signMessage(`ecs-relay-service: ${signatureExpirationTime}`),
+    expirationTime: signatureExpirationTime,
+  };
```

Comment:
Right now this only creates a single signature that expires after 5 seconds. We'll need to continuously create new signatures somewhere.